package.json

```
@@ -1,7 +1,26 @@
 {
+  "name": "purescript-elmish",
+  "license": "MIT",
+  "description": "A PureScript implementation of The Elm Architecture (TEA)",
+  "author": {
+    "name": "Fyodor Soikin",
+    "email": "fyodor@collegevine.com",
+    "url": "https://github.com/fsoikin"
+  },
+  "contributors": [
+    {
+      "name": "Daniel Gasienica",
+      "email": "daniel.gasienica@collegevine.com",
+      "url": "https://github.com/gasi"
+    },
```

Thanks! I’d like to be included with the following:

```suggestion
    {
      "name": "Daniel Gasienica",
      "email": "daniel@gasienica.ch",
      "url": "https://www.gasi.ch"
    },
```

---

Updated your info.